bin/release_tag.rb

```diff
@@ -18,7 +18,12 @@
 review = StringIO.new
 post_review = StringIO.new
 
-ManageIQ::Release.repos_for(opts).each do |repo|
+# Move 'manageiq' repo to the end of the list
+repos = ManageIQ::Release.repos_for(opts)
+miq, repos = repos.partition { |r| r.github_repo == "ManageIQ/manageiq" }
+repos += miq
```

Comment:
```suggestion
repos = ManageIQ::Release.repos_for(opts)
repos = repos.partition { |r| r.github_repo != "ManageIQ/manageiq" }.flatten
```

Talked with @kbrock and we think this is a bit cleaner